torch/csrc/jit/codegen/cuda/ir_interface_nodes.h

```diff
@@ -300,6 +300,16 @@ class TORCH_CUDA_API TensorView : public Val {
   //
   TensorView* rFactor(const std::vector<int>& axes);
 
+  // Create a TensorView before the original tensor. A common use case is to
+  // read original tensor into shared memory or local registers. Analogous to
+  // TVM Cache_Write
+  TensorView* cache_before();
+
+  // Create a TensorView after the original tensor. A common use case is to
+  // write results into shared memory or local registers before moving to global
+  // memory. Analogous to TVM Cache_Read
+  TensorView* cache_after();
+
```

Comment:
Are these comments correct? Shouldn't they be swapped?

Reply:
I switched them back.